apps/L1E/instagram/instagram-backend/jest.setup.ts

```
@@ -0,0 +1,8 @@
+// Suppress console.error during tests to clean up output
+beforeAll(() => {
+	jest.spyOn(console, 'error').mockImplementation(() => {});
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)